snmp/datadog_checks/snmp/snmp.py

```diff
@@ -226,7 +226,7 @@ def fetch_results(self, config, all_oids, bulk_oids):
         for result_oid, value in all_binds:
             metric, indexes = config.resolve_oid(result_oid)
             results[metric][indexes] = value
-        self.log.debug('Raw results: %s', results)
+        self.log.debug('Raw results: %s', OIDPrinter(results, False))
```

Comment:
Lemme suggest passing a kwarg here to more easily make sense of what `False` corresponds to…

```suggestion
        self.log.debug('Raw results: %s', OIDPrinter(results, with_values=False))
```

(We should do this for other invocations of `OIDPrinter` IMO.)